providers/instagram-graph.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "instagram-graph",
```

Comment:
Not sure if it is necessary to distinguish "Instagram Graph API" from the old [legacy API](https://www.instagram.com/developer/) given the latter one has been definitely deprecated.

Reply:
I changed provider name to `instagram`.